examples/shell/cmd_btp.cpp

```diff
@@ -0,0 +1,192 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <core/CHIPCore.h>
+
+#if CONFIG_NETWORK_LAYER_BLE > 0
+
+#include <platform/CHIPDeviceLayer.h>
+#include <shell/shell.h>
+#include <support/CHIPArgParser.hpp>
+#include <support/CHIPMem.h>
+#include <support/CodeUtils.h>
+
+using namespace chip;
+using namespace chip::Shell;
+using namespace chip::Platform;
+using namespace chip::DeviceLayer;
+using namespace chip::Logging;
+using namespace chip::ArgParser;
+
+static chip::Shell::Shell sShellDeviceSubcommands;
+
+int cmd_btp_help_iterator(shell_command_t * command, void * arg)
+{
+    streamer_printf(streamer_get(), "  %-15s %s\n\r", command->cmd_name, command->cmd_help);
+    return 0;
+}
+
+int cmd_btp_help(int argc, char ** argv)
+{
+    sShellDeviceSubcommands.ForEachCommand(cmd_btp_help_iterator, NULL);
+    return 0;
+}
+
+int cmd_btp_adv(int argc, char ** argv)
+{
+    CHIP_ERROR error  = CHIP_NO_ERROR;
+    streamer_t * sout = streamer_get();
+
+    if (argc == 0)
+    {
+        ExitNow(error = CHIP_ERROR_INVALID_ARGUMENT);
+    }
+
+    if (strcmp(argv[0], "start") == 0)
+    {
+        streamer_printf(sout, "Starting BLE advertising");
+        // start advertising
```

Comment:
Should this comment and below (line 68 and tthe ones in scan) be prefixed with 'TODO' ?